packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -96,6 +96,7 @@ class MediaQueryData {
     this.alwaysUse24HourFormat = false,
     this.accessibleNavigation = false,
     this.invertColors = false,
+    this.highContrastContent = false,
```

Comment:
Is the "content" suffix really needed?

Reply:
hmm `contentHighContrast` ?

Reply:
I just meant that qualifying highContrast with "content" doesn't really add much. I meant to suggest just calling the property highContrast.